content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm/kubernetes.md

```diff
@@ -14,9 +14,9 @@ further_reading:
 
 ## Overview
 
-<div class="alert alert-warning">Single Step Instrumentation for Kubernetes is GA for Agent versions 7.64+, and in Preview for Agent versions <=7.63.</div>
+<div class="alert alert-warning">Single Step Instrumentation for Kubernetes is GA for Agent versions 7.64+</div>
```

Comment:
What happened to the support for Agent versions 7.63 and lower? If Singe Step Instrumentation is GA for all Agent versions, than you should remove this whole div.